kore/app/exec/Main.hs

```diff
@@ -375,7 +381,11 @@ parserInfoModifiers =
 main :: IO ()
 main = do
     options <- mainGlobal parseKoreExecOptions parserInfoModifiers
-    Foldable.forM_ (localOptions options) mainWithOptions
+    Foldable.forM_ (localOptions options) $ \o ->
+        catch (mainWithOptions o) $
+            \(Goal.GoalException (e :: SomeException) p) -> do
```

Comment:
The type annotation here should not be necessary. Please do not use single-letter variable names.

```suggestion
            \(Goal.GoalException someException p) -> do
```